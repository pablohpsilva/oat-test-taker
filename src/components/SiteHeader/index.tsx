import React from 'react'

const SiteHeader = () => <div>hey! I'm siteheader!</div>

export default SiteHeader
