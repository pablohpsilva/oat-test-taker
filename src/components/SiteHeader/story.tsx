import React from 'react'
import { storiesOf } from '@storybook/react'

import { SiteHeader } from '@/components'
import readme from './readme.raw.md'

const SiteHeaderComp = () => <SiteHeader />

const opts = {
    notes: { markdown: readme }
}

storiesOf('Components|SiteHeader', module)
    .addDecorator(storyFn => <div>{storyFn()}</div>)
    .add('Complex example', () => <SiteHeaderComp />, opts)
