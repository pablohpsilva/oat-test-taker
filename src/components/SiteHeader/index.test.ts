import React from 'react'
import SiteHeader from './'

test('should render the SiteHeader component', async () => {
    const wrapper = shallow(<SiteHeader />)

    expect(wrapper.first().hasClass('SiteHeader')).toMatchSnapshot()
    expect(wrapper).toMatchSnapshot()
})
