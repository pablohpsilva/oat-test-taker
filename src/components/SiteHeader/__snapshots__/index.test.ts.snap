// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the SiteHeader component 1`] = `false`;

exports[`should render the SiteHeader component 2`] = `
<SiteHeader__StyledSiteHeader
  color="var(--color-light-0)"
  cols="auto auto auto"
  rows="1fr"
>
  <SiteHeader__StyledColumn />
  <SiteHeader__StyledColumn
    align="center"
    valign="center"
  >
    <Link
      to="/"
    >
      OAT - Test Takers
    </Link>
  </SiteHeader__StyledColumn>
  <SiteHeader__StyledColumn
    align="flex-end"
    padding="0 calc(2 * var(--dim-gutter))"
    valign="center"
  >
    <Link
      to="/import"
    >
      <Icon
        alt="import test takers"
        src={null}
      />
    </Link>
  </SiteHeader__StyledColumn>
</SiteHeader__StyledSiteHeader>
`;
